scripts/download_incr_DB.py

```diff
@@ -140,7 +140,16 @@ def GetAllObjectsFromS3(url, folderName=""):
 def GetPersistenceKey(key_url):
 	retry_counter = 0
 	while True:
-		response = requests.get(key_url, stream=True)
+		try:
+		    response = requests.get(key_url, stream=True)
+		except Exception as e:
+			print("Exception occured:" + key_url)
+			retry_counter+=1
+			if retry_counter > 3:
+				break
```

Comment:
Might be good to add `print("Failed to download: " + key_url)` here before exiting